src/components/RepositoryGoals.js

```diff
@@ -72,7 +76,18 @@ function RepositoryGoals({user}) {
           </ContextStyle>
           <Cards fitted>
             <AddRepoForm goalsId={goalsId} onGoalAdded={onGoalAdded} />
-            <ListGoals goals={repository.issues} />
+            {repository.issues.totalCount > 0 ? (
+              <ListGoals data={data} goals={repository.issues} />
+            ) : (
+              <EmptyPlaceholder>
+                <div style={{color: "grey"}}>
+                  <Octicon size="large" verticalAlign="middle" icon={getIconByName("checklist")} />
+                </div>
+                <div className="helper">
+                  No Goals created
```

Comment:
Message strings could be moved to a resource file. To help with easier i18n & g11n

Reply:
I think the i18n should be considered in its feature/PR, for now, I think it is acceptable to keep the tech debt in. As much as I would consider translations for this project, making the app usable is a higher priority.

Reply:
And something to consider: people have to speak english to contribute to OSS.